host/tests/test/manual_test.rs

```diff
@@ -0,0 +1,93 @@
+use crate::common::{complete_proof_request, v2_assert_report, Client};
+use raiko_core::interfaces::{ProofRequestOpt, ProverSpecificOpts};
+use raiko_host::server::api;
+use raiko_tasks::TaskStatus;
+use serde_json::json;
+
+/// This test is used to manually test the proof process. Operator can use this to test case to
+/// simplly test online service.
+///
+/// To run this test, you need to set the `RAIKO_TEST_MANUAL_PROVE_ENABLED` environment variable to `true`, and
+/// specify the parameters via environment variables.
+///
+/// ```bash
+/// RAIKO_TEST_MANUAL_PROVE_ENABLED=true \
+/// RAIKO_TEST_MANUAL_PROVE_API_VERSION=v2 \
+/// RAIKO_TEST_MANUAL_PROVE_NETWORK=taiko_mainnet \
+/// RAIKO_TEST_MANUAL_PROVE_PROOF_TYPE=native \
+/// RAIKO_TEST_MANUAL_PROVE_BLOCK_NUMBER=656443 \
+/// RAIKO_TEST_MANUAL_PROVE_RAIKO_RPC_URL=https://rpc.raiko.xyz \
+/// cargo test --test test_manual_prove -- --ignored
+/// ```
+#[tokio::test]
+#[ignore]
+pub async fn test_manual_prove() {
```

Comment:
also test_manual_aggregation()?? maybe after you refactory the agg v3 api, as current aggregation flow is not good enough.